frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/AbstractEnhancedGraphTable.java

```diff
@@ -0,0 +1,39 @@
+package com.virtuslab.gitmachete.frontend.ui.api.table;
+
+import javax.swing.table.AbstractTableModel;
+
+import org.checkerframework.checker.guieffect.qual.UI;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+
+public abstract class AbstractEnhancedGraphTable extends BaseGraphTable {
```

Comment:
Wrt. naming... maybe `BaseRefreshableGraphTable` + `RefreshableGraphTable`?

Reply:
Also... maybe this can be just an interface? :thinking:

Reply:
IMHO `Refreshable` is not enough. I'd stay with `Enhanced`

Reply:
Yes, `AbstractEnhancedGraphTable` can be interface

Reply:
So, rename to `BaseEnhancedGraphTable`